plugins/pip/PopupWindow.vala

```diff
@@ -97,7 +97,7 @@ public class Gala.Plugins.PIP.PopupWindow : Clutter.Actor {
 
         set_position (x_position, y_position);
 
-        close_button = new Gala.CloseButton (scale) {
+        close_button = new Gala.CloseButton () {
```

Comment:
Don't we have to set the scale here somewhere? Or is it always guaranteed to be one?